site-pages/all_products/products.css

```diff
@@ -0,0 +1,63 @@
+.page-container{
+    display: flex;
+    gap: var(--content-padding);
+    margin-bottom: 2.5rem;
+}
+.product-container{
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    width: 77%;
+}
+.product-cards-container{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
By default ,flex Direction is row ,so this can be ignored

Reply:
Oh yes, okay! Will remove it.